tests/TestCase/Heartbeat/SensorTest.php

```diff
@@ -73,4 +73,37 @@ public function testGetStatus()
         $this->assertEquals('2017-03-30 12:45:37', $status->getLastExecuted());
         $this->assertEquals(1, $status->getSeverity());
     }
+
+    /**
+     * Tests whether the check result was fetched from cache or by running the check now
+     *
+     * @return void
+     * @covers Sensor\Status::wasCheckCached
+     */
+    public function testWasCheckCached()
+    {
+        Chronos::setTestNow('2017-03-30 12:45:37');
+        $sensorName = 'Dummy Sensor';
```

Comment:
May be "Cached Sensor" is better?

Reply:
To be honest, I just used the same name that other tests used. Changed name in a commit moving the test from your previous comment.